src/platform/logging/BUILD.gn

```diff
@@ -0,0 +1,91 @@
+# See Project CHIP LICENSE file for licensing information.
+
+import("//build_overrides/build.gni")
+import("//build_overrides/chip.gni")
+import("//build_overrides/pigweed.gni")
+
+import("${chip_root}/src/platform/device.gni")
+
+config("includes") {
+  include_dirs = [ "include" ]
+}
+
+source_set("vlog") {
+  public = [ "include/logging/LogV.h" ]
+  public_configs = [ ":includes" ]
+}
+
+if (chip_enable_openthread) {
+  import("//build_overrides/openthread.gni")
+
+  if (chip_device_platform == "linux" || chip_device_platform == "Darwin") {
+    import("//build_overrides/ot_br_posix.gni")
+  }
+}
+
+simple_log_platforms = [
+  "cc13x2_26x2",
+  "darwin",
+  "efr32",
+  "esp32",
+  "k32w",
+  "linux",
+  "nrfconnect",
```

Comment:
Could we have `nrfconnect` replaced with `zephyr` here and `nrfconnect` platform depending on `logging:zephyr`? Logging.cpp for nrfconnect uses only Zephyr API, so it can be made reusable by other Zephyr-based platforms in the future.

Reply:
I had that oringally, however I saw that platform is called 'nrfconnect' still and making directory name the same with platform name makes my looping code simpler. We can always rename and split out the library, however I would prefer to do that as a followup.

Reply:
ok, fine with me!